### PR TITLE
Added tests for all `yii\db\QueryBuilder::resetSequence` implementations

### PR DESCRIPTION
**Pull request by boboldehampsink**
In #13582 we added this to the MSSQL driver, and agreed to add these for all drivers.

| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #13582

### CODE REVIEW

**Review by samdark — APPROVED** [review]

**Review by dynasource — APPROVED** [review]
thanks!

**Comment by samdark** [comment]
https://travis-ci.org/yiisoft/yii2/jobs/203429387#L589

**Comment by boboldehampsink** [comment]
I actually found a bug in the SQLite implementation that I have now fixed.

**Comment by boboldehampsink** [comment]
Okay, locally this test runs just fine. As you can see in the sqlite fixture the table "item" does get created. Maybe there is something wrong with the travis setup?

**Comment by samdark** [comment]
Maybe. I'll re-run locally...

**Comment by samdark** [comment]
@dynasource now it works as expected. Thanks!

**Comment by samdark** [comment]
Merged. Thanks!

File: framework/db/sqlite/QueryBuilder.php [hunk 1]
```diff
@@ -134,13 +134,13 @@ public function resetSequence($tableName, $value = null)
                 $key = reset($table->primaryKey);
                 $tableName = $db->quoteTableName($tableName);
                 $value = $this->db->useMaster(function (Connection $db) use ($key, $tableName) {
-                    return $db->createCommand("SELECT MAX('$key') FROM $tableName")->queryScalar();
+                    return $db->createCommand("SELECT MAX($key) FROM $tableName")->queryScalar();
```
Comment (cebe) [inline framework/db/sqlite/QueryBuilder.php]: key should be quoted the same way as it was fixed in pgsql #13582

File: framework/db/sqlite/QueryBuilder.php [hunk 2]
```diff
@@ -134,13 +134,13 @@ public function resetSequence($tableName, $value = null)
...
                 });
             } else {
                 $value = (int) $value - 1;
             }
             try {
-                $db->createCommand("UPDATE sqlite_sequence SET seq='$value' WHERE name='{$table->name}'")->execute();
+                return "UPDATE sqlite_sequence SET seq='$value' WHERE name='{$table->name}'";
             } catch (Exception $e) {
```
Comment (cebe) [inline framework/db/sqlite/QueryBuilder.php]: this try and catch does not make sense anymore if the command is not executed here.

File: framework/db/sqlite/QueryBuilder.php (hunk 1 above)
Comment (boboldehampsink) [inline framework/db/sqlite/QueryBuilder.php]: As in `{$key}`

File: framework/db/sqlite/QueryBuilder.php (hunk 2 above)
Comment (boboldehampsink) [inline framework/db/sqlite/QueryBuilder.php]: Indeed

**Comment by boboldehampsink** [comment]
@cebe I submitted a new PR to deal with the issues you've found: #13755

**Pull request merged**